gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/src/AuthorizationService.js

```diff
@@ -67,6 +81,20 @@ fluid.defaults("gpii.oauth2.authorizationService", {
         getAuthForAccessToken: {
             func: "{dataStore}.findAuthByAccessToken"
                 // accessToken
+        },
+        getCredentialClientByAccessToken: {
+            func: "{dataStore}.findCredentialClientByAccessToken"
+                // accessToken
+        },
+        grantAccessTokenForOAuth2CredentialClientId: {
+            funcName: "gpii.oauth2.authorizationService.grantAccessTokenForOAuth2CredentialClientId",
+            args: ["{dataStore}", "{arguments}.0", "{arguments}.1"]
+                // clientId, scope
+        },
+        savePrefs: {
+            funcName: "gpii.oauth2.authorizationService.savePrefs",
+            args: ["{preferencesDataSource}", "{arguments}.0", "{arguments}.1"]
+                // accessToken, preferences, callback function
```

Comment:
The comment describing the invoker arguments does not match the args array -- 3 values vs 2. From the corresponding function signature, it looks like the arguments are: req and callback. I think it's better if the invoker in this component is not dependent on the incoming HTTP request structure. Please extract the relevant parts from the request and then pass to this invoker.